framework/source/class/qx/bom/client/Event.js

```diff
@@ -32,21 +32,19 @@ qx.Bootstrap.define("qx.bom.client.Event",
     /**
      * Checks if touch events are supported.
      *
-     * @internal
      * @return {Boolean} <code>true</code> if touch events are supported.
      */
-    getTouch : function() {
+    getTouch : function () {
```

Comment:
I don't think this is usual code styling in qooxdoo. A short investigation shows that there is not space between the keyword function an the brackets in the rest of the code.